.vib/cassandra/goss/goss.yaml

```
@@ -0,0 +1,31 @@
+file:
+  {{ .Vars.persistence.mountPath }}:
+    exists: true
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /opt/bitnami/cassandra/conf/cassandra.yaml:
+    exists: true
+    filetype: file
+    mode: "0644"
+    group: root
+    contains:
+      - /num_tokens.*{{ .Vars.cluster.numTokens }}/
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: {{ .Vars.serviceAccount.create }}
+    filetype: directory
+    mode: "3777"
+command:
+  {{- $user := .Vars.dbUser.user }}
+  {{- $password := .Vars.dbUser.password }}
+  {{- $nodes := .Vars.cluster.seedCount }}
+  {{- $port := .Vars.containerPorts.cql }}
+  {{- $keyspace := printf "test_%s" (randAlpha 5) }}
+  manage-keyspace:
+    exec: "cqlsh -u {{ $user }} -p {{ $password }} -e 'CREATE KEYSPACE {{ $keyspace }} WITH REPLICATION = { $$class$$: $$SimpleStrategy$$, $$replication_factor$$ : 3 };' cassandra-0.cassandra-headless {{ $port }}{{ range $e, $i := until $nodes }} && cqlsh -u {{ $user }} -p {{ $password }} -e 'USE {{ $keyspace }};' cassandra-{{ $i }}.cassandra-headless{{ end }}"
```

I think you are missing the port in the last command:

```suggestion
    exec: "cqlsh -u {{ $user }} -p {{ $password }} -e 'CREATE KEYSPACE {{ $keyspace }} WITH REPLICATION = { $$class$$: $$SimpleStrategy$$, $$replication_factor$$ : 3 };' cassandra-0.cassandra-headless {{ $port }}{{ range $e, $i := until $nodes }} && cqlsh -u {{ $user }} -p {{ $password }} -e 'USE {{ $keyspace }};' cassandra-{{ $i }}.cassandra-headless {{ $port }}{{ end }}"
```

---

Indeed, Cassandra uses `port = 9042` when it is not specified and I missed adding it explicitly.